apps/src/aichat/views/modelCustomization/compare-models-dialog.module.scss

```diff
@@ -38,11 +38,7 @@ div.modelComparisonDialog {
   justify-content: flex-end;
 }
 
-.overflowYHidden {
-  overflow-y: hidden;
-}
-
-.overflowYScroll {
+.modelDescriptionContainer {
   overflow-y: scroll;
```

Comment:
Have you considered `auto`, so that the scrollbars won't show if they aren't needed?

Reply:
Interestingly, the scrollbars don't show with `scroll` on Chrome (screencast vid above). But I'll go ahead and update to `auto` to be safe. Thanks!